internal/fs/fs.go

```diff
@@ -5,8 +5,21 @@ import (
 	"os"
 	"os/user"
 	"strconv"
+	"strings"
 )
 
+func ExpandHomeDir(path string) string {
+	if len(path) == 0 {
+		return path
+	}
+	if path[0] == '~' {
+		// TODO: Improve compat with other OSes
+		path = strings.Replace(path, "~", "${HOME}", 1)
```

Comment:
How about using a function from the standard library `os.UserHomeDir` that returns the user's home dir and works on main OSs? It essentially does what you've done here for macos/linux. Please see how it's used in a couple other places in the code base: https://github.com/psviderski/uncloud/blob/f85b5dd01d60d30cb153c6af2b9677e60668f613/cmd/uncloud/main.go#L22-L28

It would be nice to migrate other places to use `ExpandHomeDir` eventually.

Reply:
Thanks, resolved.